### ISSUE TITLE
TLS 1.3 not supported

### ISSUE THREAD

**Issue by ogmueller** [dmke/mattercheck#1]
When calling mattercheck on our installation, I get the following error message:

```
[Jun  6 11:31:50] ERROR mattercheck: could not check instance error=Get https://mattermost.tld.com: remote error: tls: protocol version not supported url=https://mattermost.tld.com
```

We like to keep security high and therefore want to use TLS 1.3 only. Mattermost is working fine with it, but mattercheck seems not to.

Would it be possible to use more modern libraries to do your connection?

Of course we could also allow TLS 1.2, but I guess decreasing security for a tool, that should check the security, is a bit counterproductive.

**Comment by dmke** [comment]
Hi, thanks for your report, and sorry for the late reply.

I've built and released v0.2.7 with the latest Go toolchain, that should fix your issue. I don't have such a strict server config (I still allow TLS 1.2), so I cannot verify that it'll help. Would you like to test the [latest release](https://github.com/dmke/mattercheck/releases)?

**Comment by ogmueller** [comment]
ok, I installed version 0.2.7 with `go install github.com/dmke/mattercheck@v0.2.7` and it seems to work:

```bash
# mattercheck https://mattermost.tld.com
[Jul 18 10:26:05]  WARN mattercheck: found update url=https://mattermost.tld.com version=v7.11.0-rc2-team
[Jul 18 10:26:05]  INFO mattercheck: current Team version changelog=https://docs.mattermost.com/install/self-managed-changelog.html#release-v8-0-major-release checksum=sha256:46b44a2a6b8d7a2bad4553e40a565f1eb3e0b86d60903d97ec4d2f37f68effb2 download=https://releases.mattermost.com/8.0.0/mattermost-team-8.0.0-linux-amd64.tar.gz?src=arc version=v8.0.0-team
```

Thanks for such a fast resolution!


Side note: the readme suggest to install it using `go get -u github.com/dmke/mattercheck`, but that doesn't seem to work on "normal" environments:
```
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```
